docker/es/justfile

```diff
@@ -26,19 +26,19 @@ NO_COLOR := "\\033[0m"
     '"$(just docker/es/health {{ host }})" != "200"' \
     "Waiting for Elasticsearch to be healthy..."
 
-@check-index index="image":
+@check-index index="image" dest="":
     -curl \
       -s \
       -H 'Accept: application/json' \
       'http://localhost:50292/_cat/indices/{{ index }}' \
-    | grep -o "{{ index }}" \
+    | grep -o "{{ (if dest == "" { index } else { dest }) }}" \
```

Comment:
This is necessary because `_cat/indices/<alias>` returns the name of the index the alias points to, not the alias name. This happens to work for the regular aliases because their names are included in the destination index names directly. Compare `echo "audio-init" | grep "audio"` which passes and `echo "audio-init-filtered" | grep "audio-filtered"` which does not pass.

```
$ curl http://localhost:50292/_cat/indices/audio-filtered
yellow open audio-init-filtered uHH8HFuMRnCNZVXXTP6ydA 18 1 4881 0 3.6mb 3.6mb
```